vulnerabilities/importers/apache_kafka.py

```diff
@@ -7,120 +7,177 @@
+    "CVE-2019-12399": {
+        "action": "include",
+        "2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.2.0, 2.2.1, 2.3.0": "affected",
+        "2.2.2, 2.3.1 and later": "fixed",
+        "affected_version_range": "vers:apache/2.0.0|2.0.1|2.1.0|2.1.1|2.2.0|2.2.1|!=2.2.2|2.3.0|<2.3.1",
+    },
+    "CVE-2018-17196": {
+        "action": "include",
+        "0.11.0.0 to 2.1.0": "affected",
+        "2.1.1 and later": "fixed",
+        "affected_version_range": "vers:apache/>=0.11.0.0|<2.1.1",
+    },
+    "CVE-2018-1288": {
+        "action": "include",
+        "0.9.0.0 to 0.9.0.1, 0.10.0.0 to 0.10.2.1, 0.11.0.0 to 0.11.0.2, 1.0.0": "affected",
+        "0.10.2.2, 0.11.0.3, 1.0.1, 1.1.0": "fixed",
+        "affected_version_range": "vers:apache/>=0.9.0.0|<=0.9.0.1|>=0.10.0.0|<=0.10.2.1|!=0.10.2.2|>=0.11.0.0|<=0.11.0.2|!=0.11.0.3|1.0.0|!=1.0.1|!=1.1.0",
+    },
+    "CVE-2017-12610": {
+        "action": "include",
+        "0.10.0.0 to 0.10.2.1, 0.11.0.0 to 0.11.0.1": "affected",
+        "0.10.2.2, 0.11.0.2, 1.0.0": "fixed",
+        "affected_version_range": "vers:apache/>=0.10.0.0|<=0.10.2.1|!=0.10.2.2|>=0.11.0.0|<=0.11.0.1|!=0.11.0.2|!=1.0.0",
+    },
+}
 
 
 class ApacheKafkaImporter(Importer):
+
+    GH_PAGE_URL = "https://raw.githubusercontent.com/apache/kafka-site/asf-site/cve-list.html"
+    ASF_PAGE_URL = "https://kafka.apache.org/cve-list"
+    spdx_license_expression = "Apache-2.0"
+    license_url = "https://www.apache.org/licenses/"
+
     @staticmethod
-    def fetch_advisory_page():
-        page = requests.get(GH_PAGE_URL)
+    def fetch_advisory_page(self):
+        page = requests.get(self.GH_PAGE_URL)
         return page.content
 
-    def set_api(self):
-        self.version_api = GitHubTagsAPI()
-        asyncio.run(self.version_api.load_api(["apache/kafka"]))
+    def advisory_data(self):
+        advisory_page = self.fetch_advisory_page(self)
 
-    def updated_advisories(self):
-        advisory_page = self.fetch_advisory_page()
-        self.set_api()
         parsed_data = self.to_advisory(advisory_page)
-        return self.batch_advisories(parsed_data)
+        return parsed_data
 
     def to_advisory(self, advisory_page):
         advisories = []
+
         advisory_page = BeautifulSoup(advisory_page, features="lxml")
         cve_section_beginnings = advisory_page.find_all("h2")
         for cve_section_beginning in cve_section_beginnings:
-            cve_id = cve_section_beginning.text.split("\n")[0]
+            # This sometimes includes text that follows the CVE on the same line -- sometimes there is a carriage return, sometimes there is not
+            # cve_id = cve_section_beginning.text.split("\n")[0]
+            # This is superior, gets only the cve id and no following text.
+            cve_id = cve_section_beginning.get("id")
+
             cve_description_paragraph = cve_section_beginning.find_next_sibling("p")
+
+            stripped_cve_description_paragraph = str(cve_description_paragraph.get_text())
```

Comment:
What about something simpler?
```suggestion
            description = str(cve_description_paragraph.get_text())
```

Reply:
I agree @pombredanne -- simpler is better.  👍